playwright/src/main/java/com/microsoft/playwright/CDPSession.java

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.playwright;
+
+import com.google.gson.JsonElement;
```

Comment:
We were hesitant committing to any particular json implementation in the public API in the past, but I believe it's well justified in this case to proceed with Gson.